docs/_entities/account_migration.md

```diff
@@ -0,0 +1,43 @@
+---
+title: AccountMigration
+---
+
+This entity is sent when a person changes their diaspora* ID (e.g. when a user migration from one to another pod happens).
+
+## Properties
+
+| Property    | Type                         | Description                                                                          |
+| ----------- | ---------------------------- | ------------------------------------------------------------------------------------ |
+| `author`    | [diaspora\* ID][diaspora-id] | The diaspora\* ID of the closed account.                                             |
+| `person`    | [Profile][profile]           | New profile of a person                                                              |
+| `signature` | [Signature][signature]       | Signature that validates original and target diaspora* IDs with the new key of person |
```

Comment:
Can you specify that the signature signs the String "AccountMigration:old-diaspora-id@example.org;new-diaspora-id@example.com". But probably not here in the table, I think a "Signature"-section below is better for that